.github/workflows/rust.yml

```diff
@@ -180,6 +180,36 @@ jobs:
         run: rustup toolchain install stable
       - name: Install Protobuf Compiler
         run: sudo apt-get install -y protobuf-compiler
+      - name: Run tests (excluding doctests and datafusion-cli)
+        env:
+          RUST_BACKTRACE: 1
+        run: |
```

Comment:
datafusion-cli is excluded here now

I also expanded out this list of tests to make it easier to read